File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString().split('\n');
let [N, M] = input[0].split(" ").map(i => +i);
let arr = Array.from(new Array(N), (x,i) => i + 1);

for (let i = 1; i <= M; i++) {
  let numArr = input[i].split(' ').map(n=>parseInt(n));
    let indexA = numArr[0] - 1;
    let indexB = numArr[1] - 1;
    let valueA = arr[indexA];
    arr[indexA] = arr[indexB];
    arr[indexB] = valueA;
}
console.log(arr.join(' '));

File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.py
import sys
input = sys.stdin.readline
N, M = list(map(int, input().split(" ")))
arr = [i+1 for i in range(N)]

for _ in range(M):
    i, j = list(map(int, input().split(" ")))
    k = arr[i-1]
    l = arr[j-1]
    arr[i-1] = l
    arr[j-1] = k
for l in range(N):
    print(arr[l], end=" ")